src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,70 @@
+package view;
+
+import domain.Dealer;
+import domain.Player;
+import domain.Players;
+import domain.Result;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class OutputView {
+    private static final String INITIAL_DISTRUIBUTE_MESSAGE = "%s와 %s에게 2장을 나누었습니다." + System.lineSeparator();
+    private static final String DEALER_DISTRIBUTE_MESSAGE = "%s 16이하라 한장의 카드를 더 받았습니다." + System.lineSeparator();
+    private static final String DEALER_CARDS_RESULT_MESSAGE = "%s 카드: %s - 결과: %d" + System.lineSeparator();
+    private static final String PLAYER_CARDS_RESULT_MESSAGE = "%s카드: %s - 결과: %d" + System.lineSeparator();
+    private static final String DEALER_RESULT_MESSAGE = "%s: %d승 %d무 %d패" + System.lineSeparator();
+    private static final String PLAYER_RESULT_MESSAGE = "%s: %s" + System.lineSeparator();
+    private static final String SPLIT_DELIMITER = ", ";
+
+
+    public void printInitialCards(Dealer dealer, Players players) {
+        System.out.println();
+        System.out.printf(INITIAL_DISTRUIBUTE_MESSAGE, dealer.getName().getName(), players.getPlayersName()
+                .stream()
+                .collect(Collectors.joining(SPLIT_DELIMITER)));
+        System.out.println(dealer.getName().getName() + ": " + dealer.getInfo().get(dealer.getName().getName()).get(0));
+        for (String key : players.getInfo().keySet()) {
+            List<String> value = players.getInfo().get(key);
+            System.out.println(key + "카드: " + value.stream().collect(Collectors.joining(SPLIT_DELIMITER)));
+        }
+    }
+
+    public void printPlayerCardsInfo(Player player) {
+        System.out.println(player.getName().getName() + "카드: " +
+                player.getCards().stream().collect(Collectors.joining(", ")));
+
+    }
+
+    public void printDistributeDealer(Dealer dealer) {
+        System.out.println();
+        System.out.printf(DEALER_DISTRIBUTE_MESSAGE, dealer.getName().getName());
+    }
+
+    public void printCardsResult(Dealer dealer, Players players) {
+        System.out.println();
+        System.out.printf(DEALER_CARDS_RESULT_MESSAGE, dealer.getName().getName(), dealer.getCards()
+                .stream()
+                .collect(Collectors.joining(SPLIT_DELIMITER)), dealer.getCardsSum());
```

Comment:
```suggestion
        System.out.printf(DEALER_CARDS_RESULT_MESSAGE, dealer.getName().getName(), String.join(SPLIT_DELIMITER, dealer.getCards()), dealer.getCardsSum());
```
intellij 는 요걸 제안 하네요.

Reply:
정정하였습니다! 수정된 방법이 더 가독성이 좋다고 생각합니다.